R/scaffold.R

```diff
@@ -42,11 +41,11 @@ scaffoldReactWidget <- function(name, npmPkg = NULL, edit = interactive()){
   message("To build JavaScript run: yarn run webpack --mode=development")
 }
 
-toDepJSON <- function(npmPkg) {
-  if (is.null(npmPkg)) {
+toDepJSON <- function(npmPkgs) {
+  if (is.null(npmPkgs)) {
     ""
   } else {
```

Comment:
I would add another check here to make sure `names(npmPkgs)` returns something sensible, like

```r
else if (!length(names(npmPkgs))) {
  stop("Must specify npm package names in the names attributes of npmPkgs")
} else {
  paste0(sprintf('"%s": "%s"', names(npmPkgs), npmPkgs), collapse = ",\n")
}
```